extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -138,6 +138,15 @@ public function loadData() {
 				foreach ( $this->mFilterGroup as $group ) {
 					if ( !empty($group) ) {
 						if ( $group == Listusers::DEF_GROUP_NAME ) {
+							/**
+							 * @see CE-1487
+							 * Until poweruser group is still being evaluated
+							 * and developed - we consider it as 'invisible'
+							 * and include it in the No group checkbox
+							 */
+							$powerUserGroupName = \Wikia\PowerUser\PowerUser::GROUP_NAME;
+							$whereGroup[] = " single_group = '{$powerUserGroupName}' ";
```

Comment:
Would be awesome, but the code uses [```implode( 'or', $whereGroup )```](https://github.com/Wikia/app/blob/dev/extensions/wikia/Listusers/SpecialListusers_helper.php#L150) instead of a proper solution...